modules/minion/tests/minion/TaskTest.php

```diff
@@ -14,6 +14,49 @@
 
 class Minion_TaskTest extends KO7_Unittest_TestCase
 {
+	protected static $initial_request;
+	protected static $base_url;
+
+	/**
+	 * Default values for the environment, see setEnvironment
+	 * @var array
+	 */
+	protected $environmentDefault =	[
+		'url.trusted_hosts' => ['www\.example\.com', 'www\.example2\.com'],
+    'site.minion_domain_name' => 'http://www.example2.com'
+	];
+
+	/**
+	 * Sets up the environment
+	 */
+	// @codingStandardsIgnoreStart
+	public function setUp() : void
+	// @codingStandardsIgnoreEnd
+	{
+		parent::setUp();
+
+		// Keep the old request object and base_url
+    // These are changed in set_domain_name and will cause other tests to
+    // fail, so they need to be stored and restored in the tearDown method.
+		static::$initial_request = Request::$initial;
+		Request::$initial = NULL;
+
+		static::$base_url = KO7::$base_url;
```

Comment:
Sorry my bad missed that one....you can also do all static variable changes via the $enviromentDefault function, they will be resetted afterwards and you can remove your reset from the tearDown function

Reply:
Oh and sorry for beeing so critical, i dont want to annoy you it's just that i have so much work with unittests currently (cause many of them do not work as they are intented to do) i am a bit detailed to those.....